jstest/testrunner/skiplist/iotjs-test-descriptor.json

```diff
@@ -2,28 +2,40 @@
   "rpi2": {
     "skip": {
       "testsets": [],
-      "testfiles": []
+      "testfiles": [
+        { "name": "test_module_dynamicload.js", "reason": "Dynamic module got removed." },
```

Comment:
This is a leftover in IoT.js indeed. I opened a PR which removes this unused testfile: https://github.com/pando-project/iotjs/pull/1885

Reply:
Okay :) Removed it from the skip list then.